src/scss/components/_heroSlide.scss

```diff
@@ -0,0 +1,40 @@
+.slider{
+  margin: 0 auto;
+  position: relative;
+  max-width: 1440px;
```

Comment:
Remove these properties: `margin`, `position` and `max-width` and add the class `component` in the template.